### DISCUSSION TITLE
Tanya bang

### DISCUSSION THREAD

**Discussion by JayFromIndonesia** [namdevel/ovoid-API#18] [Q&A] [answered]
Bang, ini ambil data dari $app->sendOtp('+628XXXXXXXXXX'), dkk nya gimana yah?

saya coba seperti ini tapi gagal :

$json = $app->sendOtp('+628XXXXXXXXXX');
$chresult = json_encode($json);
$json_result = json_decode($chresult, true);

echo $json_result; // work
echo $json_result['response_message']; // not work
echo $json_result['data']['otp']['otp_ref_id']; // not work

**Comment by namtxs** [comment]
lgsung json_decode saja gaperlu di json_encode

```php
$send_otp = $app->sendOtp('+628XXXXXXXXXX');
$json = json_decode($send_otp , true);
echo $json['data']['otp']['otp_ref_id'];

**Comment by riagie** [comment]
untuk mengetahui url yang di gunakan pada aplikasi ovo menggunakan aplikasi apa ya? untuk capture nya

**Reply by namtxs** [reply, marked as answer]
pake frida  
[Frida • A world-class dynamic instrumentation framework](https://frida.re/)
